envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt

```diff
@@ -20,17 +20,19 @@ import java.time.Duration
 import java.util.concurrent.TimeUnit
 import kotlin.random.Random
 
-sealed class EnvoyConfigFile(val filePath: String)
-object AdsAllDependencies : EnvoyConfigFile("envoy/config_ads_all_dependencies.yaml")
-object AdsCustomHealthCheck : EnvoyConfigFile("envoy/config_ads_custom_health_check.yaml")
-object FaultyConfig : EnvoyConfigFile("envoy/bad_config.yaml")
-object Ads : EnvoyConfigFile("envoy/config_ads.yaml")
-object AdsWithDisabledEndpointPermissions : EnvoyConfigFile("envoy/config_ads_disabled_endpoint_permissions.yaml")
-object AdsWithStaticListeners : EnvoyConfigFile("envoy/config_ads_static_listeners.yaml")
-object AdsWithNoDependencies : EnvoyConfigFile("envoy/config_ads_no_dependencies.yaml")
-object Xds : EnvoyConfigFile("envoy/config_xds.yaml")
+sealed class envoy1ConfigFile(val filePath: String)
+object AdsAllDependencies : envoy1ConfigFile("envoy/config_ads_all_dependencies.yaml")
+object AdsCustomHealthCheck : envoy1ConfigFile("envoy/config_ads_custom_health_check.yaml")
+object FaultyConfig : envoy1ConfigFile("envoy/bad_config.yaml")
+object Ads : envoy1ConfigFile("envoy/config_ads.yaml")
+object Envoy1Ads : envoy1ConfigFile("envoy/config_ads_envoy1.yaml")
+object Envoy2Ads : envoy1ConfigFile("envoy/config_ads_envoy2.yaml")
```

Comment:
IMO `envoy1ConfigFile` should be reverted to `EnvoyConfigFile`

Reply:
Pwned by intellij refactor